scripts/new_avl.py

```diff
@@ -0,0 +1,569 @@
+##adapted from github of Pavel Grafov: https://github.com/pgrafov
```

Comment:
I've been thinking more about how to comply with this part of the MIT license and give credit where it is due:
> The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.

We could create a [LICENSE-3RD-PARTY text file](http://softwareengineering.stackexchange.com/questions/234511/what-is-the-best-practice-for-arranging-third-party-library-licenses-paperwork) that contains the copyright and MIT licence for `python-avl-tree`.  Our readme could also mention the third party code used and point to the license file.  We can also add a header to `new_avl.py` with the copyright and MIT license.

How does that sound?  I've seen some suggestions to keep all third party code in a `third_party` subdirectory, but I don't think that is necessary here.